src/transformers/modeling_tf_utils.py

```diff
@@ -584,6 +584,37 @@ def __init__(self, config, *inputs, **kwargs):
         self.config = config
         self.name_or_path = config.name_or_path
 
+    @tf.function(
+        input_signature=[
+            {
+                "input_ids": tf.TensorSpec((None, None), tf.int32, name="input_ids"),
+                "attention_mask": tf.TensorSpec((None, None), tf.int32, name="attention_mask"),
+                "token_type_ids": tf.TensorSpec((None, None), tf.int32, name="token_type_ids"),
```

Comment:
is `token_type_ids` general enough to be here? I'm fine with `attention_mask`

Reply:
As far as I see, `token_type_ids` appears in more models than not.

Reply:
My logic behind was to take the same inputs their those given by their respective tokenizer

Reply:
If you follow the tokenizer logic (as I think you should!), then you're right to define both attention mask and token type ids here, as that's [what's done in `model_input_names`](https://github.com/huggingface/transformers/blob/master/src/transformers/tokenization_utils_base.py#L1531).

Since you're overriding that in models down the road, this change LGTM.